src/safeposix/vmmap_constants.rs

```diff
@@ -0,0 +1,32 @@
+pub const PROT_READ: i32 = 0x1; /* Page can be read.  */
```

Comment:
a lot of these are constants that will be used elsewhere, we probably want to figure out how to structure constants in general for RawPOSIX going forward